### PR TITLE
USH-4358: Set cc_breadcrumbs_enabled

### PR DESCRIPTION
**Pull request by MartinRiese**
## Product Description

Add separate property for adding breacrumbs information so we can set it separately later if we want to.

## Technical Summary

https://dimagi.atlassian.net/browse/USH-4385

## Feature Flag

PERSISTENT_MENU_SETTING

### Safety story

The property itself is not doing anything yet. Tested locally that it gets set in profile.xml.

### Automated test coverage

No

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

File: corehq/apps/app_manager/templates/app_manager/profile.xml [hunk 1]
```diff
@@ -28,7 +28,9 @@
     {% if support_email %}
     <property key="support-email-address" value="{{ support_email }}" force="true"/>
     {% endif %}
-    <property key="cc_persistent_menu" value="{{ app.persistent_menu }}" force="true"/>
+
+    <property key="cc_persistent_menu" value="{{ app.persistent_menu|yesno:"yes,no" }}" force="true"/>
```
Comment (MartinRiese) [inline corehq/apps/app_manager/templates/app_manager/profile.xml:32]: @shubham1g5 I changed this one to yes/no too so they at least use the same kind of values.

**Review by shubham1g5 — COMMENTED** [review]
should not there also be the related UI changes here to show these settings to user ?

File: corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml [hunk 2]
```diff
@@ -181,8 +181,16 @@
...
+  description: Show the persistent menu on the side of the web app.
   toggle: PERSISTENT_MENU_SETTING
   widget: bool
   default: false
   since: '2.54'
+
+- id: show_breadcrumbs
+  name: Show Breadcrumbs
+  description: Show the breadcrumbs on top of the web app.
+  toggle: PERSISTENT_MENU_SETTING
```
Comment (esoergel) [inline corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml:193]: Is this meant to be behind the persistent menu feature flag?

Comment (MartinRiese) [reply corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml:193]: Yes, I did not want to create yet another FF. And removing the breadcrumbs does not really make sense without showing the persistent menu

**Review by shubham1g5 — APPROVED** [review]
@MartinRiese Code looks good to me here, Quick question - Does these new properties need FP changes to be propogated to Web Apps ? Or Web Apps have a way to read these directly from HQ ?

File: corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml [hunk 3]
```diff
@@ -181,7 +181,15 @@
...
 - id: persistent_menu
   name: Show Persistent Menu
-  description: Show a persistent menu instead of breadcrumbs.
+  description: Show the persistent menu on the side of the web app.
+  toggle: PERSISTENT_MENU_SETTING
+  widget: bool
+  default: false
+  since: '2.54'
+
+- id: show_breadcrumbs
```
Comment (orangejenny) [inline corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml:190]: What would you think of incorporating `web_apps` into the name? This matters less if the plan is to eventually deprecate this setting.

File: corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml (hunk 2 above)
Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml:193]: Would a feature flag negate the need for an app setting?

File: corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml (hunk 3 above)
Comment (orangejenny) [reply corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml:190]: (I'm ambivalent, I see this is part of the new web apps grouping of settings, I'm just thinking about people who are reading code or app files.)

**Comment by orangejenny** [comment]
> Does these new properties need FP changes to be propogated to Web Apps ?

Yeah, this also needs a formplayer change, web apps doesn't have generic access to the profile settings.

I also agree with Shubham's earlier point about the UI changes. It's not good practice to release user-facing changes that don't do anything. I realize this is behind a custom feature flag, but even though it has a limited audience, users should be able to trust that when they change a setting, the system changes.

File: corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml (hunk 2 above)
Comment (shubham1g5) [reply corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml:193]: I think we don't really need an app setting for peristent menu but enabling ff should be enought to enable them for the domain. 

Although we might want fine grained control on breadcrumbs as product wants to have ability to have both breadcrumbs and persistent menu enabled for a project to facilititate smooth user transition from breadcrumbs to peristent Menu.

**Pull request merged**